test/dotcom/alerts/subway_test.exs

```diff
@@ -158,10 +167,12 @@ defmodule Dotcom.Alerts.SubwayTest do
       effects =
         Enum.find(groups, fn {group, _effects} -> group == random_group end) |> Kernel.elem(1)
 
-      random_effect = Enum.random(effects)
+      {random_effect, random_severity} = Enum.random(effects)
+
+      alert = %Alert{effect: random_effect, severity: random_severity}
```

Comment:
nit: why not `Factories.Alerts.Alert.build(:alert, effect: random_effect, severity: random_severity)`?

Reply:
Just because we don't need any other attributes.